ios/FluentUI/People Picker/PeoplePicker.swift

```diff
@@ -360,7 +363,11 @@ open class PeoplePicker: BadgeField {
 
     public override func textFieldDidEndEditing(_ textField: UITextField) {
         super.textFieldDidEndEditing(textField)
-        isShowingPersonaSuggestions = false
+        guard let showPersonaSuggestions = delegate?.shouldKeepShowingPersonaSuggestionsOnEndEditing?() else {
+            isShowingPersonaSuggestions = false
```

Comment:
Same here. I think you could just do the following:

`isShowingPersonaSuggestions = delegate?.shouldKeepShowingPersonaSuggestionsOnEndEditing?() ?? false`

Reply:
Its failing though and compiler asks to add nil check hence added it with guard. As this is optional method and its implementation isn't always available